packages/browser/src/extensions/exception-autocapture/index.ts

```diff
@@ -139,9 +138,10 @@ export class ExceptionObserver {
         this.startIfEnabled()
     }
 
-    captureException(errorProperties: ErrorProperties) {
+    captureException(input: unknown, hint?: { handled?: boolean; syntheticException?: Error }) {
+        const errorProperties = this._instance.exceptions.buildProperties(input, hint)
         const posthogHost = this._instance.requestRouter.endpointFor('ui')
-
+        // @ts-expect-error: Do we still need this?
         errorProperties.$exception_personURL = `${posthogHost}/project/${
```

Comment:
We can probably remove this... we have the person info elsewhere so not really sure why anyone would use this